packages/flutter_test/test/live_binding_test.dart

```diff
@@ -0,0 +1,17 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+// This file is for testings that require a `LiveTestWidgetsFlutterBinding`
+void main() {
+  LiveTestWidgetsFlutterBinding();
+  testWidgets('Input PointerAddedEvent', (WidgetTester tester) async {
+    await tester.pumpWidget(const MaterialApp(home: Text('Test')));
+    await tester.pump();
+    final TestGesture gesture = await tester.createGesture();
+    await gesture.addPointer();
```

Comment:
nit: consider adding a comment here or at the top of the test explaining that this mimics the start of a gesture as seen on a device.